hummingbot/connector/exchange/injective_v2/account_delegation_script.py

```diff
@@ -0,0 +1,84 @@
+import asyncio
+
+from pyinjective.async_client import AsyncClient
+from pyinjective.composer import Composer
+from pyinjective.constant import Network
+from pyinjective.transaction import Transaction
+from pyinjective.wallet import PrivateKey
+
+# Values to be configured by the user
+NETWORK = Network.testnet()  # Select the correct network: mainnet, testnet, devnet, local or custom
+GRANT_EXPIRATION_IN_DAYS = 365
+GRANTER_ACCOUNT_PRIVATE_KEY = ""
+GRANTER_SUBACCOUNT_INDEX = 0
+GRANTEE_PUBLIC_INJECTIVE_ADDRESS = ""
+MARKET_IDS = []
+# List of the ids of all the markets the grant will include, for example:
+# MARKET_IDS = ["0x0511ddc4e6586f3bfe1acb2dd905f8b8a82c97e1edaef654b12ca7e6031ca0fa"]  # noqa: mock
```

Comment:
It would be possible, but would require the script to initialize an instance of the connector. I don't think it would be a good idea for the following reasons:
- It would make the script, intended to be a very straight forward and simple script, more complex.
- permissions delegation between accounts is something delicate. The user should be fully aware of what they are doing from the Injective protocol point of view. They should be explicitly configuring the script with the markets they want to include instead of trusting any translation/transformation logic